Utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A module with helper functions
+ * @module utils
+ */
+
+/**
+ * Generates the topic name in which the MQTT client must publish to actuate in a device.
+ * @function generateDojotActuationTopic
+ *
+ * @param {string} tenant
+ * @param {string} deviceId
+ * @param {string} suffix
+ *
+ * @returns {string}
+ */
+const generateDojotActuationTopic = (tenant, deviceId, suffix) => `${tenant}:${deviceId}${suffix}`;
+
+/**
+ * Transforms a string into a boolean (case insensitive).
+ * @function toBoolean
+ *
+ * @param {string} value
+ *
+ * @returns {boolean}
+ */
+const toBoolean = (value) => (value && (value.toString().toLowerCase().trim() === 'true'));
+
+/**
+ * Kills the program process.
+ * @function killApplication
+ */
+const killApplication = () => process.kill(process.pid);
```

Comment:
I suggest you change the name of this function, since the [default signal](https://nodejs.org/api/process.html#process_process_kill_pid_signal) that is sent to the process is `SIGTERM`, so this function would be more coherent if it were called `terminateApplication`.

_Note that_ at certain points in the code you are using the `process.exit(1);` instruction to end the Nodejs process. It does not perform graceful shutdown. Just a point of attention.